variables.tf

```diff
@@ -0,0 +1,109 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "resource_group_id" {
+  type        = string
+  description = "The resource group name where the Hyper Protect Crypto Service instance will be created"
+}
+
+variable "region" {
+  type        = string
+  description = "The region where you want to deploy your instance."
+}
+
+variable "service_name" {
+  type        = string
+  description = "The name to give the Hyper Protect Crypto Service instance. Max length allowed is 30 chars."
+}
+
+variable "plan" {
+  type        = string
+  description = "The name of the service plan that you choose for your Hyper Protect Crypto Service instance"
+  default     = "standard"
+  validation {
+    condition     = contains(["standard"], var.plan)
+    error_message = "Only the standard plan is supported currently"
+  }
+}
+
+variable "auto_initialization_using_recovery_crypto_units" {
+  type        = bool
+  description = "Set to true if auto initialization using recovery crypto units is required"
+  default     = false
+}
+
+variable "number_of_crypto_units" {
+  type        = number
+  description = "The number of operational crypto units for your service instance"
+  default     = 2
+  validation {
+    condition     = contains([2, 3], var.number_of_crypto_units)
+    error_message = "Allowed value of number_of_crypto_units is 2 or 3"
+  }
+}
+
+variable "tags" {
+  description = "Optional list of resource tags to apply to resources created"
```

Comment:
I would be more specific: Optional list of resource tags to apply to the HPCS instance

Reply:
done